src/torrent.cpp

```diff
@@ -1384,16 +1382,25 @@ bool is_downloading_state(int const st)
 #endif
 
 #ifdef TORRENT_SSL_PEERS
-
-	bool torrent::verify_peer_cert(bool const preverified, boost::asio::ssl::verify_context& ctx)
+	bool torrent::verify_peer_cert(std::string const& expected, bool preverified, ssl::verify_context& ctx)
```

Comment:
nit: `preverified` lost its `const`. it doesn't look like it needs to be mutable.